docusaurus.config.ts

```diff
@@ -239,7 +239,7 @@ export default {
         routeBasePath: "blog",
         include: ["**/*.{md,mdx}"],
         exclude: ["**/_*.{js,jsx,ts,tsx,md,mdx}", "**/_*/**", "**/*.test.{js,jsx,ts,tsx}", "**/__tests__/**"],
-        postsPerPage: 10,
+        postsPerPage: 999,
```

Comment:
Use 'ALL' if you want to get all posts at once.

Reply:
Resolved, Thanks.